README.md

```diff
@@ -109,6 +109,17 @@ niceties like a GUI config editor.
 
 [tmux]: https://github.com/tmux/tmux
 
+**_Why is mouse reporting not working on Windows?_**
+
+Alacritty for Windows is using programming interface known as conpty to
+communicate with the programs running in the terminal. That interface supports
+mouse events on Windows 11 but not yet on Windows 10.
+
+It is possible to make Alacritty to use the updated conpty code by copying
+files named conpty.dll and OpenConsole.exe to any directory in PATH or to the
+directory where the Alacritty executable is located. Those files can be found
+in other terminal emulators for Windows.
```

Comment:
There's numerous issues with the default ConPTY. This section should be way more generic.

It's also pointlessly detailed, nobody cares what it's called or how it works, they just want a fix.

Reply:
Maybe something like that

```markdown
**_I'm on Windows and applications don't work like in other terminals?_**

Alacritty on Windows is relying on the system libraries by default, which are
quite out of date most of the time, for example your mouse might not work in
vim on Windows 10. Other applications, like Windows Terminal, ship updated
versions of the required libraries which are known to work better, however
they are not officially provided by the microsoft [yet].

Alacritty doesn't ship those libraries, but it could still load them if they are
present in your `PATH`. Some other terminal emulators for windows do build their
own copies of such libraries, so you could take those libs from them.


[yet]: https://github.com/microsoft/terminal/issues/15065

```

Reply:
I don't think we should drop the file names, `conpty.dll`  and `OpenConsole.exe`. That would make the documentation less helpful. Also, I'm not sure `OpenConsole.exe` can be called a library. Please feel free to fix my text in `README.md`.

Reply:
I've applied some of the suggestions.

Reply:
I think this is still significantly too verbose. It should be easy to explain this in a single paragraph. It's not necessary to mention potential issues at all, if people are running into problems we just need to point them at things they can try. We also don't need to document in Alacritty how other terminal emulators behave.

When it comes to how to get this to work, we don't need to really mention anything in the FAQ at all. I'd just add a section to the install.md.